### PR TITLE
Add calendar phase2

### PR DESCRIPTION
**Pull request by utkarsh-tf141**

### CODE REVIEW

**Comment by 1-Harshit** [comment]
Don't hardcode, concatenate.


---
Harshit Raj
Junior Undergraduate
CSE, IIT Kanpur
harshitraj.me<https://harshitraj.me/>
________________________________
From: Utkarsh Mishra ***@***.***>
Sent: Sunday, February 5, 2023 5:57:06 PM
To: spo-iitk/ras-backend ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [spo-iitk/ras-backend] Add calendar phase2 (PR #108)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/spo-iitk/ras-backend/pull/108

Commit Summary

  *   217a2a3<https://github.com/spo-iitk/ras-backend/pull/108/commits/217a2a305b36b9124a0349289a214ed5f3b33c1f> added phase 2 calendars
  *   0e14481<https://github.com/spo-iitk/ras-backend/pull/108/commits/0e14481c4b09d9e67069b4525d464cfd56fb9576> removed extra line

File Changes

(2 files<https://github.com/spo-iitk/ras-backend/pull/108/files>)

  *   M application/util.calendar.go<https://github.com/spo-iitk/ras-backend/pull/108/files#diff-49964806445b24779e24de757d905a124090d993d4319c7b88ddc92982503328> (8)
  *   M secret.yml.template<https://github.com/spo-iitk/ras-backend/pull/108/files#diff-ae3fc775209f2fe43e9f2ba06437a95858900b7a4fb4385e28d5f6ecf9b7f381> (2)

Patch Links:

  *   https://github.com/spo-iitk/ras-backend/pull/108.patch
  *   https://github.com/spo-iitk/ras-backend/pull/108.diff

File: application/util.calendar.go [hunk 1]
```diff
@@ -93,14 +93,20 @@ func insertCalenderApplicationDeadline(proforma *Proforma, event *ProformaEvent)
...
 func getCalenderID(rid uint) (cID string) {
-
-	if rid == 1 {
-		cID = viper.GetString("CALENDAR.CID1")
-	}
-
-	if rid == 2 {
-		cID = viper.GetString("CALENDAR.CID2")
-	}
+	cID = viper.GetString(`CALENDAR.CID` + fmt.Sprint(rid))
```
Comment (1-Harshit) [inline application/util.calendar.go:96]:
```suggestion
	cID = viper.GetString(fmt.Sprintf("CALENDAR.CID%d",rid))
```
ig this would be better

Comment (1-Harshit) [reply application/util.calendar.go:96]: and remove comments?

**Comment by 1-Harshit** [comment]
add new lines at end of file

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Review by 1-Harshit — APPROVED** [review]

**Comment by abhishekshree** [comment]
template review, GG @1-Harshit

**Pull request merged**